doc/conf.py

```diff
@@ -323,11 +339,18 @@
 # Grouping the document tree into Texinfo files. List of tuples
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
-texinfo_documents = [
-  (master_doc, 'zephyr', u'Zephyr Project Documentation',
-   author, 'Zephyr', 'One line description of project.',
-   'Miscellaneous'),
-]
+if tags.has("kernel"):
+    texinfo_documents = [
```

Comment:
Same here; we don't use the texinfo builder, do we?

Reply:
(See above)

We can probably comment these out in another PR.